database/migrations/2019_12_28_211025_create_customers_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('customers', function (Blueprint $table) {
             $table->bigIncrements('id');
-            $table->string('uid')->unique();
+            $table->string('uuid')->unique();
```

Comment:
Just in case there are some people who already have Simple Commerce installed locally, it would be good for them not to have to remigrate everything.

Would you mind moving all the rename stuff to a single migration?